### DISCUSSION TITLE
Airbus B612 font not showing in `kitty +list-fonts`

### DISCUSSION THREAD

**Discussion by coaxial** [kovidgoyal/kitty#7046] [Q&A]
I'd like to use the B612 font, for which I've obtained the files from https://github.com/polarsys/b612/tree/master/fonts/ttf. It's showing in Mint's fonts application but `+list-fonts` doesn't list it, but it doesn list other custom fonts in the same fonts directory. I've also tried with the `fonts-b612` package.

Any pointers?

```
$ kitty --version
kitty 0.32.0 created by Kovid Goyal

$ fc-list | grep -i b612
/usr/share/fonts/opentype/b612/B612-BoldItalic.otf: B612:style=Bold Italic
/usr/share/fonts/opentype/b612/B612Mono-Regular.otf: B612 Mono,B612:style=Regular
/usr/share/fonts/opentype/b612/B612-Italic.otf: B612:style=Italic
/usr/share/fonts/opentype/b612/B612-Bold.otf: B612:style=Bold
/usr/share/fonts/opentype/b612/B612-Regular.otf: B612:style=Regular
/usr/share/fonts/opentype/b612/B612Mono-Italic.otf: B612 Mono,B612:style=Italic,Regular
/usr/share/fonts/opentype/b612/B612Mono-BoldItalic.otf: B612 Mono,B612:style=Bold Italic,Regular
/usr/share/fonts/opentype/b612/B612Mono-Bold.otf: B612 Mono,B612:style=Bold,Regular

$ kitty +list-fonts
Andale Mono
    Andale Mono
    Andale Mono

Anonymous Pro
    Anonymous Pro
    Anonymous Pro Bold
    Anonymous Pro Bold Italic
    Anonymous Pro Italic

Courier
    Courier Bold
    Courier Bold Italic
    Courier Italic
    Courier Regular

Courier 10 Pitch
    Courier 10 Pitch Bold
    Courier 10 Pitch Bold Italic
    Courier 10 Pitch Italic
    Courier 10 Pitch Regular

Courier New
    Courier New
    Courier New
    Courier New Bold
    Courier New Bold
    Courier New Bold Italic
    Courier New Bold Italic
    Courier New Italic
    Courier New Italic

DejaVu Sans Mono
    DejaVu Sans Mono
    DejaVu Sans Mono Bold
    DejaVu Sans Mono Bold Oblique
    DejaVu Sans Mono Oblique

FreeMono
    FreeMono
    FreeMono Bold
    FreeMono Bold Oblique
    FreeMono Oblique

Hack
    Hack Bold
    Hack Bold Italic
    Hack Italic
    Hack Regular

IPAGothic
    IPAGothic
    IPAGothic

JuliaMono
    JuliaMono Black
    JuliaMono Black
    JuliaMono BlackItalic
    JuliaMono BlackItalic
    JuliaMono Bold
    JuliaMono Bold
    JuliaMono BoldItalic
    JuliaMono BoldItalic
    JuliaMono BoldLatin
    JuliaMono BoldLatin
    JuliaMono ExtraBold
    JuliaMono ExtraBold
    JuliaMono ExtraBoldItalic
    JuliaMono ExtraBoldItalic
    JuliaMono Light
    JuliaMono Light
    JuliaMono LightItalic
    JuliaMono LightItalic
    JuliaMono Medium
    JuliaMono Medium
    JuliaMono MediumItalic
    JuliaMono MediumItalic
    JuliaMono Regular
    JuliaMono Regular
    JuliaMono RegularItalic
    JuliaMono RegularItalic
    JuliaMono RegularLatin
    JuliaMono RegularLatin
    JuliaMono SemiBold
    JuliaMono SemiBold
    JuliaMono SemiBoldItalic
    JuliaMono SemiBoldItalic

Liberation Mono
    Liberation Mono
    Liberation Mono
    Liberation Mono Bold
    Liberation Mono Bold
    Liberation Mono Bold Italic
    Liberation Mono Bold Italic
    Liberation Mono Italic
    Liberation Mono Italic

MathJax_Vector
    MathJax_Vector-Regular

MathJax_Vector-Bold
    MathJax_Vector-Bold

Mitra
    Mitra Mono

Nimbus Mono L
    Nimbus Mono L Bold
    Nimbus Mono L Regular
    Nimbus Mono L Regular Oblique

Nimbus Mono PS
    Nimbus Mono PS Bold
    Nimbus Mono PS Bold Italic
    Nimbus Mono PS Italic
    Nimbus Mono PS Regular
    NimbusMonoPS-Bold
    NimbusMonoPS-BoldItalic
    NimbusMonoPS-Italic
    NimbusMonoPS-Regular

Noto Mono
    Noto Mono

Symbols Nerd Font Mono
    Symbols Nerd Font Mono

Terminus (TTF)
    Terminus (TTF)
    Terminus (TTF) Bold
    Terminus (TTF) Bold-Italic
    Terminus (TTF) Italic

Tlwg Mono
    Tlwg Mono
    Tlwg Mono Bold
    Tlwg Mono Bold Oblique
    Tlwg Mono Oblique

Tlwg Typo
    Tlwg Typo
    Tlwg Typo Bold
    Tlwg Typo Bold Oblique
    Tlwg Typo Oblique

Ubuntu Mono
    Ubuntu Mono
    Ubuntu Mono Bold
    Ubuntu Mono Bold Italic
    Ubuntu Mono Italic
```

**Comment by kovidgoyal** [comment]
Read the kitty FAQ

On Tue, Jan 23, 2024 at 03:49:31AM -0800, Rock wrote:
> I'd like to use the B612 font, for which I've obtained the files from https://github.com/polarsys/b612/tree/master/fonts/ttf. It's showing in Mint's fonts application but `+list-fonts` doesn't list it.
> 
> Any pointers?
> 
> ```
> $ kitty --version
> kitty 0.32.0 created by Kovid Goyal
> 
> $ kitty +list-fonts
> Andale Mono
>     Andale Mono
>     Andale Mono
> 
> Anonymous Pro
>     Anonymous Pro
>     Anonymous Pro Bold
>     Anonymous Pro Bold Italic
>     Anonymous Pro Italic
> 
> Courier
>     Courier Bold
>     Courier Bold Italic
>     Courier Italic
>     Courier Regular
> 
> Courier 10 Pitch
>     Courier 10 Pitch Bold
>     Courier 10 Pitch Bold Italic
>     Courier 10 Pitch Italic
>     Courier 10 Pitch Regular
> 
> Courier New
>     Courier New
>     Courier New
>     Courier New Bold
>     Courier New Bold
>     Courier New Bold Italic
>     Courier New Bold Italic
>     Courier New Italic
>     Courier New Italic
> 
> DejaVu Sans Mono
>     DejaVu Sans Mono
>     DejaVu Sans Mono Bold
>     DejaVu Sans Mono Bold Oblique
>     DejaVu Sans Mono Oblique
> 
> FreeMono
>     FreeMono
>     FreeMono Bold
>     FreeMono Bold Oblique
>     FreeMono Oblique
> 
> Hack
>     Hack Bold
>     Hack Bold Italic
>     Hack Italic
>     Hack Regular
> 
> IPAGothic
>     IPAGothic
>     IPAGothic
> 
> JuliaMono
>     JuliaMono Black
>     JuliaMono Black
>     JuliaMono BlackItalic
>     JuliaMono BlackItalic
>     JuliaMono Bold
>     JuliaMono Bold
>     JuliaMono BoldItalic
>     JuliaMono BoldItalic
>     JuliaMono BoldLatin
>     JuliaMono BoldLatin
>     JuliaMono ExtraBold
>     JuliaMono ExtraBold
>     JuliaMono ExtraBoldItalic
>     JuliaMono ExtraBoldItalic
>     JuliaMono Light
>     JuliaMono Light
>     JuliaMono LightItalic
>     JuliaMono LightItalic
>     JuliaMono Medium
>     JuliaMono Medium
>     JuliaMono MediumItalic
>     JuliaMono MediumItalic
>     JuliaMono Regular
>     JuliaMono Regular
>     JuliaMono RegularItalic
>     JuliaMono RegularItalic
>     JuliaMono RegularLatin
>     JuliaMono RegularLatin
>     JuliaMono SemiBold
>     JuliaMono SemiBold
>     JuliaMono SemiBoldItalic
>     JuliaMono SemiBoldItalic
> 
> Liberation Mono
>     Liberation Mono
>     Liberation Mono
>     Liberation Mono Bold
>     Liberation Mono Bold
>     Liberation Mono Bold Italic
>     Liberation Mono Bold Italic
>     Liberation Mono Italic
>     Liberation Mono Italic
> 
> MathJax_Vector
>     MathJax_Vector-Regular
> 
> MathJax_Vector-Bold
>     MathJax_Vector-Bold
> 
> Mitra
>     Mitra Mono
> 
> Nimbus Mono L
>     Nimbus Mono L Bold
>     Nimbus Mono L Regular
>     Nimbus Mono L Regular Oblique
> 
> Nimbus Mono PS
>     Nimbus Mono PS Bold
>     Nimbus Mono PS Bold Italic
>     Nimbus Mono PS Italic
>     Nimbus Mono PS Regular
>     NimbusMonoPS-Bold
>     NimbusMonoPS-BoldItalic
>     NimbusMonoPS-Italic
>     NimbusMonoPS-Regular
> 
> Noto Mono
>     Noto Mono
> 
> Symbols Nerd Font Mono
>     Symbols Nerd Font Mono
> 
> Terminus (TTF)
>     Terminus (TTF)
>     Terminus (TTF) Bold
>     Terminus (TTF) Bold-Italic
>     Terminus (TTF) Italic
> 
> Tlwg Mono
>     Tlwg Mono
>     Tlwg Mono Bold
>     Tlwg Mono Bold Oblique
>     Tlwg Mono Oblique
> 
> Tlwg Typo
>     Tlwg Typo
>     Tlwg Typo Bold
>     Tlwg Typo Bold Oblique
>     Tlwg Typo Oblique
> 
> Ubuntu Mono
>     Ubuntu Mono
>     Ubuntu Mono Bold
>     Ubuntu Mono Bold Italic
>     Ubuntu Mono Italic
> ```
> 
> -- 
> Reply to this email directly or view it on GitHub:
> https://github.com/kovidgoyal/kitty/discussions/7046
> You are receiving this because you are subscribed to this thread.
> 
> Message ID: ***@***.***>

-- 
_____________________________________

Dr. Kovid Goyal 
https://www.kovidgoyal.net
https://calibre-ebook.com
_____________________________________
